copado-function/app/Deploy.js

```diff
@@ -89,7 +89,7 @@ const CONFIG = {
         destinationChildren: process.env.envVariablesDestinationChildren,
     },
     // commit
-    commitMessage: null,
+    commitMessage: 'Local dev commit message',
```

Comment:
this should be something dynamic like... "Updated BU <bu name here> (<mid here>)"
you will have to set it down below because you only get that info during the execution of run().